scripts/ovpsim_log_to_trace_csv.py

```diff
@@ -27,34 +27,35 @@
 from riscv_trace_csv import *
 
 try:
-    from ovpsim_log_to_trace_csv_vectors import *
+  from ovpsim_log_to_trace_csv_vectors import *
 except:
-    def assign_operand_vector(a,b,c,d):
-        """ stub version when no vector processing included """
-        logging.info("No OVPsim vector instruction processing included")
-        if stop_on_first_error:
-            sys.exit(-1)
-    def is_an_extension_instruction(instr):
-        if 'v' in instr[0]:
-            return True
-        return False
+  def assign_operand_vector(a,b,c,d):
+    """ stub version when no vector processing included """
+    logging.info("No OVPsim vector instruction processing included")
+    if stop_on_first_error:
+      sys.exit(-1)
+  def is_an_extension_instruction(instr):
+    if instr:
```

Comment:
I think it should be `if instr and `v` in instr[0]`

Reply:
Good point, changed.